simpledotorg/simple-server
# DHIS2 feature flag and scheduling

**Story card:** [ch3940](https://app.clubhouse.io/simpledotorg/story/3940/add-scheduled-task-to-send-dhis2-data-monthly)

## Because

DHIS2 export should be feature flagged and scheduled for export on the first of each month

## This addresses

- Adds a `dhis2_export` feature flag
- Schedules export at 4am local time on 1st of each month

## Test instructions

1. In the console: `Flipper.enable(:dhis2_export)`
2. On the command line: `whenever` (to confirm the cron job is scheduled).
3. In the console: `Flipper.disable(:dhis2_export)`
4. On the command line: `whenever` (to confirm the cron job is no longer scheduled)
